recipes/implot/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(implot REQUIRED)
```

Comment:
Please don't use `find_package` in the test package.
cmake is not supported by implot itself, so the name of the `findXX.cmake` module and the name of the `implot::implot` target is conan specific.

Just link to `${CONAN_LIBS}`.